### PR TITLE
Added contributing guidelines

### PR DESCRIPTION
**Pull request by binarymist**
Please review and provide feedback @ricekot?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,25 +6,80 @@ Contributions from community are key to making purpleteam a high quality compreh
...
+The output of eslint needs to be error free before submitting a PR. This should take place automatically as part of the git `pre-commit` on most purpleteam-lab projects.
+For example: `git commit` will run the `pre-commit` hook, which runs the `test` script before allowing commit. The `test` script has a `pretest` script which runs `npm run lint`
+
+We use npm scripts for most build tasks.
+
+#### What should you, the author of a Pull Request (PR), expect from us (purpleteam-labs Team)?
 
+* How much time (maximum) until the first feedback? 1 week
+* And following iterations? 1 week
+* This is a guideline we should normally be able to hit. If it's been more than a week and you haven't heard then please feel free to add a comment to your PR and @ mention the team (@purpleteam-labs/team-purpleteam-labs)
```
Comment (ricekot) [inline CONTRIBUTING.md:52]: guideline -> deadline?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,25 +6,80 @@ Contributions from community are key to making purpleteam a high quality compreh
...
 
+#### What we (purpleteam-labs Team), expect from you?
```
Comment (ricekot) [inline CONTRIBUTING.md:54]:
```suggestion
#### What do we (purpleteam-labs Team) expect from you?
```

**Review by ricekot — APPROVED** [review]
Looks good, just a few minor suggestions.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,25 +6,80 @@ Contributions from community are key to making purpleteam a high quality compreh
 
 Depending on your preference, you can contribute in various ways.
 
-Usually the best approach is to discus what you're thinking of or planning first. You can do this by:
+Often the best approach is to discus what you're thinking of or planning first. You can do this by:
```
[the remark this answers is not in the document]
Comment (binarymist) [reply CONTRIBUTING.md:9]: Looks good

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -6,25 +6,80 @@ Contributions from community are key to making purpleteam a high quality compreh
...
-The purpleteam-labs [Backlog/Project board](https://github.com/purpleteam-labs/purpleteam/projects/2) lists work items waiting for someone to start work on. These can be a good place to start.
+The purpleteam-labs [Backlog/Project board](https://github.com/purpleteam-labs/purpleteam/projects/2) lists work items for all of the purpleteam-labs projects waiting for someone to start work on. These can be a good place to start.
+
+You can also [open an issue](https://github.com/purpleteam-labs/purpleteam/issues/new/choose), or send a Pull Request (PR).
 
 We also have a [Roadmap](https://github.com/purpleteam-labs/purpleteam/projects/1) for Epics a little further out.
 
-You can also [open an issue](https://github.com/purpleteam-labs/purpleteam/issues/new/choose), or send a Pull Request.
+If you are OK with creating a PR that may never be accepted, then go ahead and just create the PR and use the description to provide context and motivation, as you would for an issue.
+If you want to start a discussion first or have already created an issue, once a pull request is created, we will close the issue as superseded by the pull request,
```
[the remark this answers is not in the document]
Comment (binarymist) [reply CONTRIBUTING.md:21]: Looks like this feature only affects issues and PRs in the same repository. We will often have issues in the purpleteam repo but PRs in any of the other repos, so in most cases they can't be automatically linked by the looks of it.

File: CONTRIBUTING.md (hunk 1 above)
Comment (binarymist) [reply CONTRIBUTING.md:52]: Yeah, I think deadline sounds better. Thanks.

File: CONTRIBUTING.md (hunk 2 above)
Comment (binarymist) [reply CONTRIBUTING.md:54]: Yeah cool

**Comment by binarymist** [comment]
* Should address your comments @ricekot https://github.com/purpleteam-labs/purpleteam/pull/35/commits/a504e3354d10d670e9e41adb1bea384241520cfb
* Added reference to "The sevel rules ..." https://github.com/purpleteam-labs/purpleteam/pull/35/commits/7cb59156f73657c1fa0a2742fcf2441877ab7cac

Look OK @ricekot ?

**Review by ricekot — APPROVED** [review]
Looks good. I read your comment on linking issues and PR later, that part can be updated I guess?

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ricekot) [reply CONTRIBUTING.md]: This blog post says they can: https://github.blog/2013-03-18-closing-issues-across-repositories/

**Comment by binarymist** [comment]
Rebased all commits in this PR on latest main. The only commit left to review @ricekot is "Add note about closing issues from PR" #e74516f

**Review by ricekot — APPROVED** [review]
Looks good for merging! Thanks!

**Comment by binarymist** [comment]
Squashed and merged. Thanks @ricekot

**Pull request closed without merging**